src/version.h

```diff
@@ -11,6 +11,9 @@
 
 static const int PROTOCOL_VERSION = 170002;
 
+//! Overwinter proto version
+static const int OVERWINTER_PROTO_VERSION = 170003;
```

Comment:
This will need to be bumped when we set the mainnet activation height.

Reply:
To 170004? - because we set 170003 for 1.0.15 which doesn't have a mainnet activation height?

Reply:
There needs to be a ticket or some process written down specifying this.

Reply:
@ebfull there is a ticket for the testnet bump, but you're right, we should have this documented somewhere. https://github.com/zcash/zcash/issues/2969